jinseok/app.js

```diff
@@ -94,7 +104,54 @@ app.get('/users/:userId/posts', async function (req, res, next) {
     [userId, userId]
   );
 
-  res.status(200).json(data);
+  res.status(200).json({ data: post });
+});
+
+app.put('/posts', async function (req, res, next) {
+  const { update, userId, postId } = req.body;
+
+  await dataSource.query(
+    `
+  UPDATE posts SET context= ?, updated_at=CURRENT_TIMESTAMP
+  WHERE user_id= ? AND id= ?;
+  `,
+    [update, userId, postId]
+  );
+
+  res.status(200).json({ message: 'post updated' });
+});
+
+app.delete('/users/:userid/posts/:postid', async function (req, res, next) {
+  const userId = req.params.userid;
+  const postId = req.params.postid;
```

Comment:
```suggestion
  const { userId, postId } = req.params;
```
구조분해할당을 활용하면 줄일 수 있는 코드입니다-!